scripts/obtain/uninstall/dotnet-uninstall-debian-packages.sh

```diff
@@ -14,11 +14,9 @@ fi
 
 host_package_name="dotnet-host"
 aspnetcore_runtime_package_name="^aspnetcore-runtime.*"
-aspnetcore_package_store_package_name="^aspnetcore-store.*"
```

Comment:
Why change this here?

Reply:
We're deprecating the aspnetcore-store package altogether in 2.1. Should we keep it in the 'master' version of this uninstall script anyways?

Reply:
No. I didn't read the description in the PR. This should be fine. We can merge this once it is green.

Reply:
Ok cool